ipn/ipnlocal/local.go

```diff
@@ -3117,6 +3125,12 @@ func dnsConfigForNetmap(nm *netmap.NetworkMap, prefs ipn.PrefsView, logf logger.
 	return dcfg
 }
 
+// SetTCPHandlerForFunnelFlow sets the TCP handler for funnel flows.
+// It should only be called before the LocalBackend is used.
+func (b *LocalBackend) SetTCPHandlerForFunnelFlow(h func(netip.AddrPort, uint16) func(net.Conn)) {
```

Comment:
It's non-obvious enough that I'd name src and dstPort here.

Reply:
done.